frontend/src/pages/sport items/leaguePage.js

```diff
@@ -113,18 +157,57 @@ function LeaguePage() {
             )
     }, [loading]);
 
+    const initialIcon = <FaFilter size={28} />
+
+    const [selectedModel, setSelectedModel] = useState("leagues");
+    const [filters, setFilters] = useState([]);
+    const [burgerMenu, setBurgerMenu] = useState(false)
+    const [menuIsOpen, setMenuIsOpen] = useState(false)
+    const [menuIcon, setMenuIcon] = useState(initialIcon)
+
+    useEffect(() => {
+
+        const handleResize = () => {
+            const smallScreen = window.innerWidth <= 1050
+            setBurgerMenu(smallScreen)
+        }
+
+        handleResize();
+        window.addEventListener("resize", handleResize);
+    }, []);
+
+    const handleOpenMenu = () => {
+        setMenuIsOpen(prev => !prev)
+        setMenuIcon(!menuIsOpen ? <FaTimes size={28} color="black" /> : initialIcon)
```

Comment:
MINOIR: 

Maybe we should keep all our colors and fonts (as well as their size) in some place?

We usually do that when we want to maintain proper color and font styles